jwst/cube_build/ifu_cube.py

```diff
@@ -572,13 +569,13 @@ def build_ifucube(self):
                 # --------------------------------------------------------------------------------
                 # POINTCLOUD used for skyalign and IFUalign
                 # --------------------------------------------------------------------------------
-                if self.interpolation == 'pointcloud':
+                if self.interpolation == 'pointcloud' or self.interpolation == 'driz':
```

Comment:
Python nit-pick, but could shorten the syntax here a bit by using:
`if self.interpolation in ['pointcloud', 'driz']:`